### PR TITLE
Community section on install methods.

### PR DESCRIPTION
**Pull request by satiricon**
Added the community installation section with the Arch AUR package as the only entry. Could be extended with other community provided installation methods.

### CODE REVIEW

File: tools/Documentation/installing-lanraragi/comunity.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Arch Linux
```
Comment (Difegue) [inline tools/Documentation/installing-lanraragi/comunity.md:1]: Needs a big ol' header here

```suggestion
# Community Install Packages  

## Arch Linux
```

File: tools/Documentation/installing-lanraragi/methods.md [hunk 2]
```diff
@@ -45,6 +45,12 @@ Installing from **source** is a more involved procedure, but it does put you in
 
 {% page-ref page="source.md" %}
 
+## Linux/Community: _Community provided install packages_
+
+Ready to install packages provided by voluntary mantainers or by a linux distribution itself.
+
+{% page-ref page="comunity.md" %}
```
Comment (Difegue) [inline tools/Documentation/installing-lanraragi/methods.md:52]:
```suggestion
{% page-ref page="community.md" %}
```  

I guess you also need to rename the matching file. 😁

File: tools/Documentation/installing-lanraragi/methods.md [hunk 3]
```diff
@@ -45,6 +45,12 @@ Installing from **source** is a more involved procedure, but it does put you in
 
 {% page-ref page="source.md" %}
 
+## Linux/Community: _Community provided install packages_
+
+Ready to install packages provided by voluntary mantainers or by a linux distribution itself.
```
Comment (Difegue) [inline tools/Documentation/installing-lanraragi/methods.md:50]:
```suggestion
Ready-to-install packages provided by voluntary maintainers or by a linux distribution itself.
```

**Review by Difegue — CHANGES_REQUESTED** [review]
Please also edit SUMMARY.md to add a link to your new page! Thanks a lot 🙇‍♂️

**Comment by satiricon** [comment]
@Difegue Added a new entry on SUMMARY, committed the changes you made and changed the community file name and merged with the dev branch of the repository.

File: tools/Documentation/SUMMARY.md [hunk 4]
```diff
@@ -9,6 +9,7 @@
 * [Homebrew \(Linux/macOS\)](installing-lanraragi/macos.md)
 * [Docker \(All platforms\)](installing-lanraragi/docker.md)
 * [Source Code \(Linux/macOS\)](installing-lanraragi/source.md)
+* [Community \(Linux\)](installing-lanraragi/source.md)
```
Comment (Difegue) [inline tools/Documentation/SUMMARY.md:12]:
```suggestion
* [Community \(Linux\)](installing-lanraragi/community.md)
```

**Review by Difegue — CHANGES_REQUESTED** [review]
One last fix and this is good for merging imo, thanks again for your time. 🤝

**Review by Difegue — APPROVED** [review]

**Comment by Difegue** [comment]
image

**Pull request merged**